packages/office-ui-fabric-react/src/components/HoverCard/Card.types.ts

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+
+import { DirectionalHint } from '../../common/DirectionalHint';
+import { IRefObject, IStyleFunctionOrObject } from '../../Utilities';
+import { IStyle, ITheme } from '../../Styling';
+
+/**
+ * Card common props.
+ */
+export interface ICardProps<T, TStyles, TStyleProps> extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
Similar to other comment I don't think it's obvious what T should be as a type if I use this interface